### PR TITLE
Rmzaoo/login register page

### PR DESCRIPTION
**Pull request by rmzaoo**

### CODE REVIEW

File: src/components/LoginRegisterContainer/loginRegisterContainer.tsx [hunk 1]
```diff
@@ -15,8 +19,10 @@ interface Props {
 const loginContainer = ({ type, className }: Props) => {
   const navigate = useNavigate();
   const dispatch = useDispatch();
+  const [registerPage, setRegisterPage] = useState(0);
+  const [showSuggestion, setShowSuggestion] = useState(false);
 
-  const [values, setValues] = React.useState({
+  const [values, setValues] = React.useState<any>({
```
Comment (GonVieira) [inline src/components/LoginRegisterContainer/loginRegisterContainer.tsx:25]: Any.

File: src/components/AddressContainerSuggestion/AddressContainerSuggestion.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import React from "react";
+import useSearchAddressComplete from "../../hooks/user/useSearchAddressComplete";
+import { ContainerSuggestion } from "./styled";
+
+interface AddressSearchProps {
+  addressSearched: string;
+  values: any;
+  setValues: any;
+  setShowSuggestion: any;
+}
```
[the remark this answers is not in the document]
Comment (rmzaoo) [reply src/components/AddressContainerSuggestion/AddressContainerSuggestion.tsx:11]: > Muito "any" neste ficheir 
Write comments in English, I'll see what I can do about it...

[the remark this answers is not in the document]
Comment (rmzaoo) [reply src/components/AddressContainerSuggestion/AddressContainerSuggestion.tsx:11]: Write comments in English, I'll see what I can do about it...

[the remark this answers is not in the document]
Comment (rmzaoo) [reply src/components/AddressContainerSuggestion/AddressContainerSuggestion.tsx:11]: > Muito "any" neste ficheiro.

Write comments in English, I'll see what I can do about it...

[the remark this answers is not in the document]
Comment (GonVieira) [reply src/components/AddressContainerSuggestion/AddressContainerSuggestion.tsx:11]: oui oui.

File: src/utils/cookies.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export function getCookie(cname: string) {
+  let name = cname + "=";
+  let decodedCookie = decodeURIComponent(document.cookie);
+  let ca = decodedCookie.split(";");
```
Comment (GonVieira) [inline src/utils/cookies.ts:4]: Maybe you can write a more detailed variable name.
At a first glance i don't know what "ca" is supposed to mean.

**Review by GonVieira — APPROVED** [review]
Beri Cool Beri naice.

**Review by aparra-11 — APPROVED** [review]

**Pull request merged**